CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/framework/GraphVisualAccess.java

```diff
@@ -66,6 +68,8 @@
 
     //TODO: Determine whether this field is needed
     private final int[] visualAttributes = new int[VisualProperty.values().length];
+    private final String BOOLEAN_ATTRIBUTE_STR = "boolean";
```

Comment:
Please replace this with `BooleanAttributeDescription.ATTRIBUTE_NAME` where its used.